packages/wonder-blocks-data/src/hooks/__tests__/use-cached-effect.test.js

```diff
@@ -190,13 +229,137 @@ describe("#useCachedEffect", () => {
             expect(fakeHandler).toHaveBeenCalledTimes(1);
         });
 
+        it.each(
+            Array.from(FetchPolicy.members()).filter(
+                (e) => e !== FetchPolicy.CacheOnly,
+            ),
+        )(
+            "should provide function that causes refetch with FetchPolicy.%s",
+            async (fetchPolicy) => {
+                // Arrange
+                const response = Promise.resolve("DATA1");
+                const fakeHandler = jest.fn().mockReturnValue(response);
+
+                // Act
+                const {
+                    result: {
+                        current: [, refetch],
+                    },
+                } = clientRenderHook(() =>
+                    useCachedEffect("ID", fakeHandler, {fetchPolicy}),
+                );
+                fakeHandler.mockClear();
+                await act((): Promise<mixed> => response);
+                act(refetch);
+                await act((): Promise<mixed> => response);
+
+                // Assert
+                expect(fakeHandler).toHaveBeenCalledTimes(1);
+            },
+        );
+
+        it("should throw with FetchPolicy.CacheOnly", () => {
+            // Arrange
+            const fakeHandler = jest.fn();
+
+            // Act
+            const {
+                result: {
+                    current: [, refetch],
+                },
+            } = clientRenderHook(() =>
+                useCachedEffect("ID", fakeHandler, {
+                    fetchPolicy: FetchPolicy.CacheOnly,
+                }),
+            );
+            const underTest = () => refetch();
+
+            // Assert
+            expect(underTest).toThrowErrorMatchingInlineSnapshot(
+                `"Cannot fetch with CacheOnly policy"`,
+            );
+        });
+
+        it.each(
+            Array.from(FetchPolicy.members()).filter(
+                (e) => e !== FetchPolicy.CacheOnly,
+            ),
```

Comment:
style nit: assigning this value to `AllFetchPoliciesExceptCacheOnly` would help with readability or list them out in an array like you do in the next `it.each()`.